src/components/basicstat.jsx

```diff
@@ -1,8 +1,13 @@
 import HoverInfo from './hoverinfo';
 import { useTranslation } from "react-i18next";
 
-function BasicStat({ title, information, sourceLink, value, percentage }) {
+function BasicStat({ title, information, sourceLink, value, percentage, optional }) {
```

Comment:
May be a good idea in the future to add a `defaultValue` parameter instead of `optional` and compare `value` to the default value since I believe some stats have defaults that are 100% for example, instead of 0